lib/run/summary.js

```diff
@@ -319,6 +373,9 @@ _.assign(RunSummary, {
             timings = timings && timings.timings;
             timingPhases = timings && sdk.Response.timingPhases(timings);
 
+            summary.run.timingPhases = timingPhases;
+            summary.run.verboseSession = _.findLast(_.get(o, 'history.execution.sessions'));
```

Comment:
Do you mean `_.last`?
```suggestion
            summary.run.verboseSession = _.last(_.get(o, 'history.execution.sessions'));
```